### DISCUSSION TITLE
Optimization of Large Antenna Arrays for Astronomical Applications

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#36] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project 205](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Optimization%20of%20Large%20Antenna%20Arrays%20for%20Astronomical%20Applications).

**Comment by anumishra10** [comment]
Hello, 

Can we select any Antenna and make any kind of array and optimize it ?
